jetty-util/src/main/java/org/eclipse/jetty/util/log/AbstractLogger.java

```diff
@@ -198,36 +198,63 @@ else if ("OFF".equalsIgnoreCase(levelStr))
      *            the fully qualified class name
      * @return the condensed name
      */
+    @SuppressWarnings("Duplicates")
     protected static String condensePackageString(String classname)
     {
+        // use of StringUtil.isEmpty() not necessary here, algorithm below filters whitespace already
```

Comment:
I don't think we need this comment.... but I also think you are confusing `StringUtil.isEmpty(String)` with `StringUtil.isBlank(String)`.  The implementation of the former is:
```java
    public static boolean isEmpty(String str)
    {
        return str == null || str.isEmpty();
    }
```

So it is exactly the if case below and slightly more readable.

Reply:
removed comment